tests/acceptance/features/apiSpacesDavOperation/moveByFileId.feature

```diff
@@ -229,3 +225,77 @@ Feature: moving/renaming file using file id
       | dav-path                          |
       | /remote.php/dav/spaces/<<FILEID>> |
       | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: move a file from project to personal space
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the GraphApi
+    And user "Alice" has uploaded a file inside space "project-space" with content "some data" to "textfile.txt"
+    And we save it into "FILEID"
+    When user "Alice" moves a file "/textfile.txt" into "/" inside space "Personal" using file-id path "<dav-path>"
+    Then the HTTP status code should be "502"
+    And the value of the item "/d:error/s:message" in the response about user "Alice" should be "move:error: not supported: cannot move across spaces"
+    And for user "Alice" folder "/" of the space "project-space" should contain these files:
+      | textfile.txt |
+    But for user "Alice" the space "Personal" should not contain these entries:
+      | textfile.txt |
```

Comment:
Either
```suggestion
    And for user "Alice" folder "/" of the space "project-space" should contain these files:
      | textfile.txt |
    But for user "Alice" folder "folder" of the space "project-space" should not contain these files:
      | textfile.txt |
```
OR
```suggestion
    And for user "Alice" the space "project-space" should contain these entries:
      | textfile.txt |
    But for user "Alice" the space "Personal" should not contain these entries:
      | textfile.txt |
```
IMO, Mixing up the steps is not a good idea. In this case, the former will fit as we check files.
Better to use `entries` for a mix of files and folders.

Reply:
okay

Reply:
Could you check for similar cases in other places too? Please don't apply the suggestions only where it is said.

Reply:
Okay sure